newm-server/src/test/kotlin/io/newm/server/features/arweave/ktx/SongExtTest.kt

```diff
@@ -0,0 +1,101 @@
+package io.newm.server.features.arweave.ktx
+import com.google.common.truth.Truth.assertThat
+import io.newm.server.features.song.model.MintingStatus
+import io.newm.server.features.song.model.Song
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.koin.core.context.startKoin
+import org.koin.core.context.stopKoin
+import org.koin.dsl.module
+import org.koin.test.KoinTest
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.util.UUID
+
+class SongExtTest : KoinTest {
+    companion object {
```

Comment:
@saif-software-developer Can you can eliminate this companion, to be consistent with the rest of all our tests?

Reply:
That took me to a rabbit-hole, and I was not able to get the test to run without this companion, maybe we can get together sometime tomorrow to figure it out. 
**_Method 'beforeAll' annotated with '@BeforeAll' should be static_** 


![Screenshot 2024-04-28 at 10 25 08 PM](https://github.com/projectNEWM/newm-server/assets/14129154/f11488b1-6022-4335-87b8-afd2247f2021)

Reply:
Timebox this to 10 minutes and then merge. It should not block this PR.

Reply:
I tried to refactor to match other tests, but it didn't work. Will add it to Tech Debt backlog